app/views/partners/requests/_history.html.erb

```diff
@@ -23,8 +25,24 @@
               </span>
             <% end %>
           </td>
+          <td class="p-4">
+            <% comment = partner_request.comments %>
+            <% if comment.present? %>
+              <div class="mb-2" data-bs-toggle="tooltip" data-bs-title="<%= comment %>">
+                <%= truncate(comment, length: 20) %>
+              </div>
+            <% end %>
+            <a href="mailto:<%= partner_request.requester.email %>"><%= partner_request.requester.email %></a>
+          </td>
         </tr>
       <% end %>
     </tbody>
   </table>
 </section>
+
+<script type="module">
+  $(document).ready(function() {
+    const tooltipTriggerList = document.querySelectorAll('[data-bs-toggle="tooltip"]')
+    const tooltipList = [...tooltipTriggerList].map(tooltipTriggerEl => new bootstrap.Tooltip(tooltipTriggerEl))
```

Comment:
Same suggestion as above. `map` implies you're doing something with the results of the loop, which you're not.